sycl/source/detail/scheduler/commands.hpp

```diff
@@ -493,10 +493,10 @@ class ExecCGCommand : public Command {
 
   detail::CG &getCG() const { return *MCommandGroup; }
 
-  // MEmptyCmd one is only employed if this command refers to host-task.
-  // MEmptyCmd due to unreliable mechanism of lookup for single EmptyCommand
-  // amongst users of host-task-representing command. This unreliability roots
-  // in cleanup process.
+  // MEmptyCmd is only employed if this command refers to host-task.
+  // Due to unreliable mechanism of lookup for single EmptyCommand amongst users
+  // of host-task-representing command. This unreliability roots in cleanup
```

Comment:
If I understand correctly, last two sentences of the comment should be merged.
```suggestion
  // of host-task-representing command, it requires the cleanup process.
```

Reply:
Maybe like this:
The mechanism of lookup for single EmptyCommand amongst users of host-task-representing command is unreliable. This unreliability roots in the cleanup process.

Reply:
Denis's suggestion is more clear for me. @s-kanaev are you ok with this? Can you change this?

Reply:
Done